backend/src/main/java/com/rota/api/StaffController.java

```diff
@@ -154,11 +155,11 @@ public StaffDto updateStaff(
    * @param id staff id to remove.
    * @return An updated list of active staff members.
    */
-  @DeleteMapping("/staff")
+  @DeleteMapping("/staff/{id}")
```

Comment:
:+1:

Reply:
Just curious - are there any guidelines on the use of path variables vs query parameters?

Reply:
I don't think there are guidelines, but the general practice that I've seen is that you use path variables for accessing resources, and use query parameters to filter and sort those resources.